src/value/decode.rs

```diff
@@ -312,7 +313,7 @@ mod tests {
                 }
 
                 let v: $t = $v;
-                data.write_bin_value(&myc::value::Value::from(v)).unwrap();
+                myc::value::Value::from(v).serialize(&mut data);
```

Comment:
Do we know that this will serialize using the binary protocol and not the text protocol?

Reply:
It seems so.  The implementation of `MySerialize` for `Value` seems to operate always directly in binary as far as I understand it.  See https://github.com/blackbeam/rust_mysql_common/blob/master/src/value/mod.rs#L73

Reply:
Huh, yeah, it seems that way even though it's not documented. I suppose the tests passing is a good indication that this is reliable.